tests/sparse_plink/Makefile

```diff
@@ -0,0 +1,59 @@
+
+CUDA=false
+COMPILER=intel
+
+ifeq ($(COMPILER), gcc)
+ FORTRAN=gfortran
+ FFLAGS=-O3 -ftree-vectorize -mtune=native -march=native 
+ FFLAGS+=-cpp -fopenmp 
+ LDFLAGS=-llapack -lblas -lgfortran -lm -lquadmath -fopenmp
+
+else ifeq ($(COMPILER), intel)
+ FORTRAN=ifort
+ FFLAGS=-O3 -mavx2 -mtune=native -march=native
+ FFLAGS+=-I${MKLROOT}/include -I${MKLROOT}/include/intel64/lp64 -L${MKLROOT}/lib/intel64
+ FFLAGS+=-fpp -qopenmp -qopt-matmul
+ LDFLAGS=-liomp5 -lmkl_blas95_lp64 -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core  -lpthread -lm -ldl -lstdc++
+
+endif
+
+SRC_DIR=$(PWD)/../../src
+FFLAGS+=-I$(SRC_DIR)/bindings/Fortran
+LDFLAGS+=-L$(SRC_DIR)/miraculix -L$(SRC_DIR)/bindings/Fortran -L$(SRC_DIR)/cuda
+LDFLAGS+= -Wl,-rpath=$(SRC_DIR)/miraculix -Wl,-rpath=$(SRC_DIR)/bindings/Fortran -Wl,-rpath=$(SRC_DIR)/cuda 
+LDFLAGS+=-lmiraculix -lfortran 
+
+
+ifeq ($(CUDA), true)
+LDFLAGS+=-L${CUDA_HOME}/lib64 -lcudart -lcublas -lcusolver -lcusparse -lcusolver -ldgemm_compressed_cuda
+FFLAGS+=-DCUDA
+endif
+
+
+OBJ = $(SRC:.f90=.o)
+MODS = $(OBJ:.o=.mod)
+
+LIBFORTRAN = $(SRC_DIR)/bindings/Fortran/libfortran.a
+LIBMIRACULIX = $(SRC_DIR)/miraculix/libmiraculix.a
+
+EXE = test_sparse_plink
```

Comment:
Could you please rename it to test_sparse_plink.out? This makes it easier to be compatible with the gitignore